airflow/www/static/js/dags.js

```diff
@@ -428,17 +433,42 @@ function refreshDagRuns(error, json) {
   });
 }
 
-function handleRefresh() {
+function getAllDagIds() {
```

Comment:
`getAllDagIds` and `getActiveDagIds` return an array of dag ids. Passing either function to `handleRefresh` or `refreshDagStats` let's the developer specify all / just a subset of dag ids to use in the HTTP request.